tests/phpunit/tests/file.php

```diff
@@ -32,9 +32,9 @@ function test_get_file_data() {
 		$actual = get_file_data( DIR_TESTDATA . '/themedir1/default/style.css', $theme_headers );
 
 		$expected = array(
-			'Name' => 'WordPress Default',
+			'Name' => 'ClassicPress Default',
 			'ThemeURI' => 'http://wordpress.org/',
-			'Description' => 'The default WordPress theme based on the famous <a href="http://binarybonsai.com/kubrick/">Kubrick</a>.',
+			'Description' => 'The default ClassicPress theme based on the famous <a href="http://binarybonsai.com/kubrick/">Kubrick</a>.',
```

Comment:
There is no ClassicPress theme (yet)

Reply:
This is just test data (this time pulled from `tests/phpunit/data/themedir1/default/style.css`).  As long as we change both instances, this is fine.